snuba/settings.py

```diff
@@ -15,15 +15,17 @@
 DATASET_MODE = "local"
 
 # Clickhouse Options
-# TODO: Warn about using `CLICKHOUSE_SERVER`, users should use the new settings instead.
-[default_clickhouse_host, default_clickhouse_port] = os.environ.get(
-    "CLICKHOUSE_SERVER", "localhost:9000"
-).split(":", 1)
```

Comment:
I assume you verified the CLICKHOUSE_SERVER parameter is not used anywhere.

Reply:
Yeah. Checked again just to be sure.